src/formatDate.js

```diff
@@ -1,14 +1,38 @@
 'use strict';
 
-/**
+/*
  * @param {string} date
  * @param {string[]} fromFormat
  * @param {string[]} toFormat
  *
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateArr = date.split(fromFormat[3]);
```

Comment:
It's assumed that `fromFormat[3]` is the separator, but this code will break if `fromFormat` has a different length. Instead, you should use the last element of the `fromFormat` array to be more robust.